superset-frontend/src/components/Select/styles.tsx

```diff
@@ -128,7 +128,7 @@ export const DEFAULT_STYLES: PartialStylesConfig = {
   clearIndicator: provider => [
     provider,
     css`
-      padding-right: 0;
+      padding: 4px 0 4px 6px;
```

Comment:
not necessarily related, but I noted when adding and removing the clear indicator in storybook, this moved around. This only happens in storybook, but made it more consistent nonetheless.